nimare/meta/cbma/mkda.py

```diff
@@ -202,21 +229,18 @@ def _compute_null_approximate(self, ma_maps):
         This method adds two entries to the null_distributions_ dict attribute:
```

Comment:
This doc string may be out of date now, `ma_maps` can be sparse arrays as well.

Reply:
I believe `ma_maps` aren't used by the `MKDADensity._compute_null_approximate()`. I think the `ma_maps` parameter is here just to provide compatibility when the `_compute_null_approximate()` is called in the `CBMAEstimator`. 
What would you recommend doing in this case?

Reply:
How does this look?

```
Parameters
----------
ma_maps
    Modeled activation maps. Unused for this estimator.
```

Reply:
okay, that sounds good to me for now